### PR TITLE
Fix aad sample

### PR DESCRIPTION
**Pull request by Netyyyy**
As title.

### CODE REVIEW

**Review by saragluna — APPROVED** [review]
LGTM.

File: aad/spring-cloud-azure-starter-active-directory/web-client-access-resource-server/aad-resource-server-obo/README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
-# Azure OAuth 2.0 Sample for azure-spring-boot-sample-active-directory-resource-server-obo client library for Java
+# Azure OAuth 2.0 Sample for spring-cloud-azure-starter-active-directory-resource-server-obo client library for Java
```
Comment (saragluna) [inline aad/spring-cloud-azure-starter-active-directory/web-client-access-resource-server/aad-resource-server-obo/README.md:1]: We should change this title, the title is not accurate

**Review by saragluna — CHANGES_REQUESTED** [review]

File: aad/spring-cloud-azure-starter-active-directory/web-client-access-resource-server/aad-resource-server-obo/README.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ The following steps constitute the OBO process and the client credential process
 We will prepare two applications to demonstrate the dependent calls of resources.
 
 
-- Configure `Web API B` Resource Server: [azure-spring-boot-sample-active-directory-resource-server]
+- Configure `Web API B` Resource Server: [spring-cloud-azure-starter-active-directory-resource-server]
```
Comment (saragluna) [inline aad/spring-cloud-azure-starter-active-directory/web-client-access-resource-server/aad-resource-server-obo/README.md:26]: the sample is not called `spring-cloud-azure-starter-active-directory-resource-server` and we don't have such library, should we improve these namings？

File: aad/spring-cloud-azure-starter-active-directory/web-client-access-resource-server/aad-resource-server-obo/README.md [hunk 3]
```diff
@@ -127,5 +127,5 @@ client Credential response success.
 [Azure portal]: https://portal.azure.com/
 
 [Resource server access other resources usage]: https://github.com/Azure/azure-sdk-for-java/blob/azure-spring-boot_3.6.0/sdk/spring/azure-spring-boot-starter-active-directory#resource-server-access-other-resources-usage
-[azure-spring-boot-sample-active-directory-resource-server]: ../aad-resource-server
+[spring-cloud-azure-starter-active-directory-resource-server]: ../aad-resource-server
```
Comment (saragluna) [inline aad/spring-cloud-azure-starter-active-directory/web-client-access-resource-server/aad-resource-server-obo/README.md:130]: same here

File: aad/spring-cloud-azure-starter-active-directory/web-client-access-resource-server/aad-resource-server-obo/README.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
-# Azure OAuth 2.0 Sample for azure-spring-boot-sample-active-directory-resource-server-obo client library for Java
+# Azure OAuth 2.0 Sample for Azure AD Spring Boot Starter Resource Server Obo client library for Java
 
 ## Key concepts
-[Resource server access other resources usage] is an extension scenario of the *azure-spring-boot-sample-active-directory-resource-server* sample. Similarly, this sample illustrates how to protect a Java web API by restricting access to its resources to authorized accounts, and the restricted resource will access other restricted resource, such as Graph API and Custom API.
+[Resource server access other resources usage] is an extension scenario of the *aad-resource-server* sample. Similarly, this sample illustrates how to protect a Java web API by restricting access to its resources to authorized accounts, and the restricted resource will access other restricted resource, such as Graph API and Custom API.
```
Comment (jialigit) [inline aad/spring-cloud-azure-starter-active-directory/web-client-access-resource-server/aad-resource-server-obo/README.md:4]: [One resource server accesses another resource server] is another usage scenario for the xxx lib.

The main point here is that we should explain what is "OBO" as the title denotes.

And, before we continue the sample, we need to list the infrastructures and its configurations. (and How-To, this is the prerequisites that have to be done before runing the app. ).  Along this, we explain the main concepts and demonstrate some diagrams just like the below section does.

Comment (saragluna) [reply aad/spring-cloud-azure-starter-active-directory/web-client-access-resource-server/aad-resource-server-obo/README.md:4]: Good point, but I think it's not in this PR's scope, this pr is to fix some obvious errors in the sample project. @jialigit could you create the pr with the changes you propose here?

**Review by saragluna — APPROVED** [review]

**Pull request merged**